networking/multiple_networks/primary_networks/about-user-defined-networks.adoc

```diff
@@ -24,6 +24,10 @@ Support for the Localnet topology on both primary and secondary networks will be
 
 Unlike a network attachment definition (NAD), which is only namespaced scope, a cluster administrator can use a UDN to create and define additional networks that span multiple namespaces at the cluster level by leveraging the `ClusterUserDefinedNetwork` custom resource (CR). Additionally, a cluster administrator or a cluster user can use a UDN to define additional networks at the namespace level with the `UserDefinedNetwork` CR. 
 
+The following diagram shows tenant isolation that a cluster administrator created by defining a `ClusterUserDefinedNetwork` (CR) for each tenant. This network configuration allows a network to span across many namespaces. In the diagram, `udn-1` selects `namespace-1` and `namespace-2`; `udn-2` selects `namespace-3` and `namespace-4`. A tenant acts as a disconnected network that is isolated from other tenants' networks. Pods from a namespace can communicate with pods in another namespace only if those namespaces exist in the same tenant network.
+
```

Comment:
Would you want to say disconnected network udn1 is across two namespaces, namespace-1 and namespace-2 while second disconnected network udn2 is across two namespaces, namepace-3 and namespace-4. It is just language, explanation of diagram makes sense.

Reply:
Hi @asood-rh . I added "disconnected network" but Surya specifically wanted to emphasize "selects" as the verb. Is this OK with you?